### ISSUE TITLE
When will the next release be?

### ISSUE THREAD

**Issue by andrster** [cucumber/godog#381]
Sorry for asking, it's just there is a security issue in 0.11 related to nextcloud.

**Comment by lonnblad** [comment]
Hi @andrster, no need to be sorry.
Can you help me with more information around the security issue?

BR
Fredrik

**Comment by andrster** [comment]
https://nextcloud.com/security/advisory/?id=NC-SA-2021-005. this should provide more info. but Nextcloud Server prior to 20.0.6 is vulnerable to reflected cross-site scripting (XSS) due to lack of sanitization in OC.Notification.show.

**Comment by lonnblad** [comment]
Hi @andrster, maybe I'm missing something, but I'm not familiar with Nextcloud and I don't understand how a security issue with Nextcloud Server has anything to do with godog.

A question on our slack community asked about a security issue with "Cobra", are those related?

**Comment by sergiughf** [comment]
Hey @lonnblad, not sure about the nextcloud security issue but there is another one detected by [whitesource](https://www.whitesourcesoftware.com/):

```
Whitesource has detected the usage of a vulnerable library github.com/cucumber/gherkin-go/v11-358b64f0b76efbd07590532f6f02f4667d28f9f9
```

**Comment by lonnblad** [comment]
> Hey @lonnblad, not sure about the nextcloud security issue but there is another one detected by [whitesource](https://www.whitesourcesoftware.com/):
> 
> ```
> Whitesource has detected the usage of a vulnerable library github.com/cucumber/gherkin-go/v11-358b64f0b76efbd07590532f6f02f4667d28f9f9
> ```

Hi @sergiughf, thanks!! Do you know if that is solved in a later version of gherkin-go?

**Comment by sergiughf** [comment]
I honestly don't know:) as I'm not able to run whitesource manually on demand, it's automated and the check reruns only after a PR merge.

**Comment by lonnblad** [comment]
ok, I will check with the gherkin-go maintainers :)

**Comment by sergiughf** [comment]
Thank you @lonnblad ! Much appreciated:)

**Comment by lonnblad** [comment]
@sergiughf I created a new issue #382 for the whitesource gherkin-go issue.

**Comment by mpkorstanje** [comment]
Closed by https://github.com/cucumber/godog/issues/382#issuecomment-799235959
